packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -147,6 +147,11 @@ const InlineMediaIndicator = styled.div`
         bottom: 0;
       }
       `}
+  ${({ displayImage, dir }) =>
+    !displayImage &&
+    (dir === 'ltr'
+      ? `float:left; position: relative !important;`
+      : `float:right; position: relative !important;`)}
```

Comment:
The problem here is that a bit higher we are setting `position: absolute` within a `@media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN})`. So it seems that the only two ways for me to assign `position: relative` for the mediaIndicator is either to have `!important` or to assign that style within a media query for all breakpoints so that it takes precedence. Do you think I should add an additional media query instead of `!important` here or is it equally bad practice?

Reply:
Actually, since `mediaIndicator` is now within a headline, this whole snippet of code got deleted. No more `!important`.